README.md

```diff
@@ -4,26 +4,48 @@ use-cardano is a react context, hook, and set of components that makes interacti
 
 The package leverages [lucid](https://github.com/spacebudz/lucid) for transaction building and off-chain code to interact with smart contracts. For more information on that, visit the [lucid documentation](https://lucid.spacebudz.io/).
 
+This package uses blockfrost as the Cardano node provider by default. You can sign up here with a free or paid subscription: [blockfrost](https://blockfrost.io/)
+
+The following are the current blockfrost proxyUrl's:
+
+```
+"https://cardano-testnet.blockfrost.io/api/v0"
+"https://cardano-preview.blockfrost.io/api/v0"
+"https://cardano-preprod.blockfrost.io/api/v0"
+"https://cardano-mainnet.blockfrost.io/api/v0"
+```
+
```

Comment:
```suggestion

```
"https://cardano-testnet.blockfrost.io/api/v0"
"https://cardano-preview.blockfrost.io/api/v0"
"https://cardano-preprod.blockfrost.io/api/v0"
"https://cardano-mainnet.blockfrost.io/api/v0"
```

```

let's remove these lines because they are not proxy urls, but rather blockfrosts own API urls (see PR comment)